easybuild/easyconfigs/p/PyTorch/PyTorch-1.8.0-foss-2020b.eb

```diff
@@ -0,0 +1,73 @@
+name = 'PyTorch'
+version = '1.8.0'
+
+homepage = 'https://pytorch.org/'
+description = """Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+PyTorch is a deep learning framework that puts Python first."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/pytorch',
+        'repo_name': 'pytorch',
+        'tag': 'v%(version)s',
+        'recursive': True,
```

Comment:
@branfosj Any concerns here w.r.t. reproducibility? Or are the submodules "locked" to a particular commit anyway?

Reply:
They are all locked to specific commits - see https://github.com/pytorch/pytorch/tree/v1.8.0/third_party and subdirectories. The only issue we'd have is if PyTorch reused the tag - then we'd get a different download (with, potentially, a different set of items in the `third_party` directory).

Reply:
Hm, a downside of this I see is that `--fetch` likely doesn't work, i.e. a full offline install fails, or does EB handle that?
Also no checksums...
BTW: There is a script in framework to create the sources list out of a valid git checkout (must have `git submodule update` done)

Reply:
`--fetch` works (so long as you do not hit https://github.com/easybuilders/easybuild-framework/issues/3619).